src/Nethermind/Nethermind.Specs/ChainSpecStyle/ChainSpecLoader.cs

```diff
@@ -225,6 +227,7 @@ chainSpec.Parameters.Eip1283DisableTransition is null
         chainSpec.LondonBlockNumber = chainSpec.Parameters.Eip1559Transition;
         chainSpec.ShanghaiTimestamp = chainSpec.Parameters.Eip3651TransitionTimestamp;
         chainSpec.CancunTimestamp = chainSpec.Parameters.Eip4844TransitionTimestamp;
+        chainSpec.PragueTimestamp = chainSpec.Parameters.PragueTransitionTimestamp;
```

Comment:
Do we need to set PragueTransitionTimestamp? Can we rely on one of the EIP included in Pectra?